README.md

```diff
@@ -0,0 +1,28 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio"
+
+> Here you will see my portfolio of works and projects I've been involved with. This is just a sample of what I can acheive.
+
+You can access my portifolio through this [**link**](https://tiago-lelinski-marin.github.io/myPortfolio/)!
```

Comment:
- [ ] As the to the previous reviewer your documentation should be more professional as it is considered as a sample of what is inside the whole project. So, I suggest you give the working link as the one provided here is not working. Once you fixed the issue your project will be complete.

- [ **OPTIONAL** ] And if you decide to add it, make sure it is placed in its appropriate section. Ir is supposed to be in the **Live Demo** section. You can take a look at the Microverse [template](https://github.com/microverseinc/readme-template) and 
- [ **OPTIONAL** ] I suggest you add the other important section in your documentation where you can place some hints about someone who wants to get the project locally. Like an example below:

  ## Getting Started

   To get a local copy up and running follow these simple example steps:

    ### Prerequisites
    - You can add here tools needed to be used when someone wants to get the project locally and use it. 
      e.g: IDE(Vscode, Sublime,etc), Git, etc.
 
    ### Setup
       - Processus on how to clone or download the Zip file, how to access the cloned directory,...
 
    ### etc

Reply:
Hi there!

First of all, thanks so much, I will do all the optional things you suggest, this review process is really making me improve my code and make it clearer and more professional, even clearer to me!

About the link, it is working but the website URL is based on the main, and on the main branch there's nothing yet!

Reply:
Will I need to resubmit again? Should I change the link anyway? (I'll but in the section of live demo like you said before)
I kinda need to keep on working on the next part!

Reply:
@tresorsawasawa